### ISSUE TITLE
[A11y] Accessibility Bug on Dropdown in Safari, options are read out followed by "you are currently on a text element, inside of a list box."

### ISSUE THREAD

**Issue by Xander667** [OfficeDev/office-ui-fabric-react#6823]
### Please provide a reproduction of the bug in a codepen:
 
To repo go to the Fabric demo component sites and open any dropdown on Safari with VoiceOver turned on. As you tap up or down and move through the dropdown options wait for the voicerover to read everything.

### Bug Report 
 
 
- __Package version(s)__: Even the latest I could find.
- __Browser and OS versions__: (fill this out if relevant)
 
#### Priorities and help requested: This is a Pri1 accessibility bug for us
 
Are you willing to submit a PR to fix? (Yes, No) Potentially yes, but no idea how to fix.
 
Requested priority: (Blocking, High, Normal, Low)  Blocking/High?
 
Products/sites affected: (if applicable) portfolio.microsoft.com next version will have more ui with the dropdowns. Current exposed on portfolios.officeppe.com 
 
#### Describe the issue:
Open a page with a dropdown on Safari browser 
Turn on voicerover (the screen reader)
Tab down to the options and then using up and down keys navigate through the dropdown options
 
#### Actual behavior:
After reading out option information, reader reads, "You are currently on a text element, inside of a list box"
#### Expected behavior:
Same as other browsers, just reading out the option as a listitem

**Comment by KevinTCoughlin** [comment]
@Xander667 which version of Safari? Perhaps there was a regression with the release of Mojave? If not, strange it would only surface in Safari.

Thank you for the report.

**Comment by cliffkoh** [comment]
Hi Xander, because behavior of screen readers can vary wildly due to factors completely beyond our control (including bugs/varying behavior between different versions of the same screen reader, as well as with the browser), our commitment in Fabric is that we will be compliant with W3C accessibility standards. We are only able to take action if you can identify a specific way that we are not adhering to the standards.

Please refer to https://github.com/OfficeDev/office-ui-fabric-react/wiki/Accessibility-Troubleshooting for more information on our processes.

**Comment by Xander667** [comment]
@KevinTCoughlin  thanks for the reply. I'm not sure if this is a regression or normal behavior. I am using version 9.1.2 of Safari when I see this. 

@cliffkoh  When I debugged the issue I noticed within the dropdown, Button elements are used for the dropdown options. When I change the aria-role on those Buttons from "option" to "listitem" on Safari the secondary description is not longer there. 
With regards to why it'll be an issue, screen reader user will get confused with the incorrect control type being announced for the interactive elements, if it announces, "this is a text element".  When role is "listitem" the voiceover reads,
"You are currently on a group inside of a list box. To interact with items in this group press ....." as the secondary text read which is more accurate and useful.

Thanks,
Xander
